schema/src/BT01/BT01CreditClass.yaml

```diff
@@ -0,0 +1,155 @@
+id: https://framework.regen.network/schema/BTCreditClass
+name: BTCreditClass
+prefixes:
+  linkml: https://w3id.org/linkml/
+  rfs: https://framework.regen.network/schema/
+  schema: http://schema.org/
+imports:
+  - linkml:types
+default_curi_maps:
+  - semweb_context
+default_prefix: rfs
+default_range: string
+
+classes:
+  BT01CreditClass:
+    class_uri: rfs:BT01CreditClass
+    slots:
+      - approvedMethodologies
+      - coBenefits
+      - creditGenerationMethod
+      - creditingTerm
+      - description
+      - ecosystemType
+      - eligibleActivities
+      - geographicApplicability
+      - name
+      - permanencePeriod
+      - primaryImpact
+      - sourceRegistry
+      - url
+  Methodology:
+    class_uri: rfs:Methodology
+    description: Details about a specific methodology.
+    attributes:
+      identifier:
+        description: Unique identifier for the methodology.
+      name:
+        description: Name of the methodology.
+      url:
+        description: URL to the methodology document.
+      version:
+        description: Version number of the methodology.
+  MethodologyList:
+    class_uri: rfs:MethodologyList
+    description: List of methodologies for credit generation.
+    attributes:
+      itemListElement:
+        range: Methodology
+        multivalued: true
+        description: "List of methodology items."
```

Comment:
I don't think we need this intermediary class. I see this was added to try and model https://schema.org/ItemList

Instead, I think we can just reference `Methodology` with `multivalued: true`

Reply:
I've made this change